src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+/**
+ * Implementation of Task. Each Todo comes with a description.
+ */
+public class Todo implements Task {
+
+    protected String desc;
+    protected boolean isMarked;
+    protected String type;
+
+    public Todo(String params){
+        this.desc = params;
+        isMarked = false;
+        type = "T";
```

Comment:
Considering how `type` will never change to any other value, should this be declared as a constant instea?